repo.go

```diff
@@ -110,6 +111,24 @@ func parseRepo(defaultSchema, defaultHost, defaultOrganization, name string) (re
 	return r, nil
 }
 
+func (r repo) isGithubRemote() (ok bool) {
+	cmd := xexec.Fmt("git remote -v")
+	out, err := cmd.Output()
+	if err != nil {
+		flog.Error("Unable to check repo remotes")
+		return false
+	}
+
+	o := strings.Split(string(out), "\n")
+
+	for _, url := range o {
+		if strings.HasPrefix(url, "https://github.com") || strings.HasPrefix(url, "git@github.com") {
```

Comment:
We should at least know the `username/repo` of the repository, so I think we should check that the URL matches exactly against what we expect it to be. For example, for Sail it should be `https://github.com/cdr/sail.git` or `git@github.com:cdr/sail.git`.